module-services/service-cellular/src/ServiceCellularPriv.cpp

```diff
@@ -550,6 +552,52 @@ namespace cellular::internal
         owner->csqTimer.start();
     }
 
+    void ServiceCellularPriv::initUSSDHandler()
+    {
+        ussdHandler->onOpenPushSession = [this]() -> bool {
+            auto channel = owner->cmux->get(CellularMux::Channel::Commands);
+            if (channel == nullptr) {
+                LOG_ERROR("No cmux channel provided!");
+                return false;
+            }
+            return channel->cmd(at::AT::CUSD_OPEN_SESSION).code == at::Result::Code::OK;
```

Comment:
I'd recommend for clarity's sake moving the command result into a variable and then comparing it in the return statement :P

Reply:
done on each occurence